src/log.cpp

```diff
@@ -0,0 +1,182 @@
+#include "log.h"
+
+namespace bpftrace {
+
+std::string logtype_str(LogType t)
+{
+  switch (t)
+  {
+    // clang-format off
+    case LogType::debug   : return "DEBUG";
+    case LogType::info    : return "INFO";
+    case LogType::warning : return "WARNING";
+    case LogType::error   : return "ERROR";
+    // clang-format on
+    default:
+      std::cerr << "Log type not found." << std::endl;
```

Comment:
"Not found" reads kinda funny. How about this instead:
```suggestion
      std::cerr << "Invalid log type." << std::endl;
```